__tests__/app.tests.js

```diff
@@ -356,4 +355,105 @@ describe("nc news app", () => {
         });
     });
   });
+  describe("PATCH /api/articles/:article_id", () => {
+    test("status: 200, responds with an object containing a key of 'article' and a value of an object", () => {
+      return request(app)
+        .patch("/api/articles/1")
+        .send({ inc_votes: 1 })
+        .expect(200)
+        .then(({ body }) => {
+          expect(Object.keys(body)[0]).toBe("article");
+          expect(body.article).toBeInstanceOf(Object);
+        });
+    });
+    test("status: 200, responds with an object with the correct keys and values types", () => {
+      return request(app)
+        .patch("/api/articles/1")
+        .send({ inc_votes: 1 })
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.article).toMatchObject({
+            article_id: expect.any(Number),
+            title: expect.any(String),
+            body: expect.any(String),
+            votes: expect.any(Number),
+            topic: expect.any(String),
+          });
+        });
+    });
```

Comment:
Although I like the amount of testing you're going into.
This test is really not needed since the next test covers all of this in much more detail